code/game/mecha/mech_bay.dm

```diff
@@ -22,6 +25,22 @@
 	recharging_mecha = null
 	return ..()
 
+
+/turf/simulated/floor/mech_bay_recharge_floor/attackby(obj/item/C as obj, mob/user as mob, params)
+	if(!C || !user) // I don't know what these do, they're leftovers from floor.dm
+		return 1
+	if(..()) // Ditto
+		return 1
+	if(intact && istype(C, /obj/item/weapon/screwdriver)) //Can't be a crowbar, because otherwise the attackby() for floor.dm intercepts the call, I think
+		to_chat(user, "<span class='danger'>You disassemble the mech bay recharge station.</span>")
+		new /obj/item/stack/sheet/plasteel(src, 2)

```

Comment:
The mechbay turf doesn't have builtin_tile (or it does? I'm not sure), so can't be crowbar'd normally. I either need to set that flag, or have the attackby override.
I don't know what best practice is. Should I have the make_tile override anyway?